README.md

```diff
@@ -113,7 +113,9 @@ For a more information, checkout our [API reference](https://docs.svix.com).
 #### Building the current commit
 
 This project uses [goreleaser](https://github.com/goreleaser/goreleaser/).
+
  1) Install [go](https://golang.org/doc/install).
+ 2) Install [snapcraft](https://snapcraft.io/docs/installing-snapcraft).
  2) Install goreleaser via the steps [here](https://goreleaser.com/install/).
  3) Build current commit via `goreleaser release --snapshot --skip-publish --rm-dist`.
```

Comment:
Fix the numbering. :)
I know it's fine in markdown, though it should be fine for cleartext too.